test/integration/concierge_tls_spec_test.go

```diff
@@ -701,7 +701,9 @@ func performKubectlApply(t *testing.T, resourceName string, yamlBytes []byte) (s
 
 	t.Cleanup(func() {
 		t.Helper()
-		require.NoError(t, exec.CommandContext(t.Context(), "kubectl", "delete", "--ignore-not-found", "-f", yamlFilepath).Run())
+		ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
```

Comment:
should this be `context.Background()` or `t.Context()`? Although it probably does not matter.